spinalcordtoolbox/image.py

```diff
@@ -1398,20 +1398,20 @@ def concat_warp2d(fname_list, fname_warp3d, fname_dest):
     warp3d = np.zeros([nx, ny, nz, 1, 3])
 
     for iz, fname in enumerate(fname_list):
-        img = nib.load(fname)
-        warp2d = np.asanyarray(img.dataobj)
+        img = Image(fname)
+        warp2d = np.asanyarray(img.data)
```

Comment:
I think `Image` guarantees that `img.data` is an actual numpy array, so this should be a no-op. Let's remove it for clarity.
```suggestion
        warp2d = img.data
```